rebasehelper/tests/conftest.py

```diff
@@ -59,3 +61,17 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mocked_spec_object(monkeypatch, spec_attributes):
+    if 'save' not in spec_attributes:
+        spec_attributes['save'] = lambda: None
```

Comment:
Ok, but why not simply do `spec.save = lambda: None` after creating `spec`?

Reply:
But of course using `monkeypatch.setattr()`.

Reply:
Actually, is `monkeypatch.setattr()` even necessary? Looking at the implementation, it doesn't to anything special except remembering the old value, which we don't need here.

Reply:
Okay, lets just use `setattr()`, that should suffice in this case.